### PR TITLE
Multi script

### PR DESCRIPTION
**Pull request by FelixWeis**
CleanShot 2023-05-07 at 09 16 04@2x

h/t @Evertype

### CODE REVIEW

**Comment by FelixWeis** [comment]
Some more suggestions by ChatGPT

> 1. Traditional Chinese: 預期碑文 (yùqī bēiwén)
>    - 預期 (yùqī): anticipated
>    - 碑文 (bēiwén): inscription
> 
> 2. Hindi (Devanagari): अपेक्षित अभिलेख (apekṣit abhilekh)
>    - अपेक्षित (apekṣit): anticipated
>    - अभिलेख (abhilekh): inscription
> 
> 3. Mongolian (traditional Mongolian script): ᠲᠥᠷᠥᠯᠥᠵᠦ ᠪᠢᠴᠢᠭ (törölzü bichig)
>    - ᠲᠥᠷᠥᠯᠥᠵᠦ (törölzü): anticipated, expected
>    - ᠪᠢᠴᠢᠭ (bichig): inscription
> 
> 4. Arabic: تسجيل متوقع (tasjil mutawaqqe')
>    - تسجيل (tasjil): inscription
>    - متوقع (mutawaqqe'): anticipated
> 
> 5. Japanese (Kanji): 予想される碑文 (Yosō sareru hibun)
>    - 予想される (Yosō sareru): anticipated
>    - 碑文 (hibun): inscription
> 
> 6. Russian (Cyrillic): ожидаемая надпись (ozhidayemaya nadpis')
>    - ожидаемая (ozhidayemaya): anticipated
>    - надпись (nadpis'): inscription
> 
> 7. Korean (Hangul): 기대되는 묘각 (gidaedoeneun myogak)
>    - 기대되는 (gidaedoeneun): anticipated
>    - 묘각 (myogak): inscription
> 
> Please note that the translations may not be perfect or literal, but they convey the general meaning of "anticipated inscription" in each language.

File: src/app.rs [hunk 1]
```diff
@@ -222,8 +221,14 @@ pub fn App(cx: Scope) -> impl IntoView {
             <main class="flex-1 overflow-y-auto">
               <div class="mx-auto max-w-7xl px-4 pt-8 sm:px-6 lg:px-8">
                 <div class="flex">
-                  <h1 class="flex-1 text-2xl font-bold text-gray-900 dark:text-gray-100">
-                    "Live unconfirmed inscriptions"
+                  <h1 class="flex-2 text-2xl font-bold text-gray-900 dark:text-gray-100">
```
Comment (sectore) [inline src/app.rs:224]: nit: Move `text-2xl font-bold text-gray-900 dark:text-gray-100` to `class` of parent `div` to avoid duplication.

**Review by sectore — CHANGES_REQUESTED** [review]
In case `i18n` might be a nice-to-have feature in `live` it's recommended to use a more general solution to handle/switch languages. By searching for `i18n` at Leptos Discord there are some discussions about how to do that with Leptos - I'm going to summarize it in another issue.

**Comment by FelixWeis** [comment]
i18n will be more than just nice-to-have. Especially given our theme of inscriptions. There is a lot to explore and the Unicode standard, made possible by [Michael Everson](https://twitter.com/evertype) and others by enabling representation to forgotten languages and scripts.

We can use the fact that ancient scripts used different css `writing-mode`. horizontal-rtl is the default in the west now but others.
https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode

That would enable a "frame" around the anticipated inscriptions with 2 red ⭕️  in each corner ⭕️  The RTL english/latin script, The hebrew/arab/persian script from LTR. And e.g. traditional chines letterson the left top to bottom and another top-to-bottom script on the right. Or any other sensible combination.

**Comment by FelixWeis** [comment]
closing in favor of #126

**Pull request closed without merging**